src/customPropTypes.js

```diff
@@ -121,9 +121,9 @@ export const surahType = PropTypes.shape({
 export const juzType = PropTypes.shape({
   id: PropTypes.number.isRequired,
   juzNumber: PropTypes.number.isRequired,
-  nameArabic: PropTypes.string.isRequired,
-  nameSimple: PropTypes.string.isRequired,
-  verseMapping: PropTypes.arrayOf(PropTypes.object)
+  nameArabic: PropTypes.string,
+  nameSimple: PropTypes.string,
+  verseMapping: PropTypes.object
```

Comment:
`verseMapping` should be required attribute, and we can remove `nameArabic ` and  `nameSimple `.